README.md

```diff
@@ -8,6 +8,7 @@ This is an interactive tutorial; you will need a Minikube cluster running locall
 1. [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
 1. `minikube start`
 1. `minikube addons enable heapster`
+1. `minikube addons enable metrics-server`
 
 You will also need the metrics API installed if it is not already:
```

Comment:
I was going to delete the instructions about needing the metrics API installed but, I did need to `kubectl apply` a deployment from the [metrics-server](https://github.com/kubernetes-incubator/metrics-server) project (on top of enabling the metrics-server addon).